.github/workflows/playwright.yml

```
@@ -0,0 +1,27 @@
+# name: Playwright Tests
```

actionlint (1.7.8)

workflow is empty

(syntax-check)